src/tests/tracing/eventpipe/common/IpcHeader.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Tracing.Tests.Common
+{
+    internal class IpcHeader
```

Comment:
This could use more comments to clarify what is the format and expectation of IpcHeader, I'm not sure of it myself or where the documentation is yet.

Reply:
The documentation is here: https://github.com/dotnet/diagnostics/blob/main/documentation/design-docs/ipc-protocol.md